lib/tilt/jbuilder.rb

```diff
@@ -103,9 +103,10 @@ def evaluate(scope, locals, &block)
     private
     def set_locals(locals, scope, context)
       view_path = options[:view_path]
-      scope.send(:instance_variable_set, '@_jbuilder_view_path', view_path)
-      scope.send(:instance_variable_set, '@_jbuilder_locals', locals)
-      scope.send(:instance_variable_set, '@_tilt_data', data)
+      scope.instance_exec(binding) {
+        @_jbuilder_view_path = view_path
+        @_jbuilder_locals = locals
+      }
```

Comment:
Unless you feel like you have a good reason not to would you mind adding `data` back in? Removing it seems unrelated to this PR and I'd rather not risk introducing a bug.